java/client/test/org/openqa/selenium/testing/drivers/ExternalDriverSupplier.java

```diff
@@ -102,7 +102,7 @@ public WebDriver get() {
     Optional<Class<? extends Supplier<WebDriver>>> supplierClass = getDelegateClass();
     if (supplierClass.isPresent()) {
       Class<? extends Supplier<WebDriver>> clazz = supplierClass.get();
-      logger.info("Using delegate supplier: " + clazz.getName());
+      logger.finest("Using delegate supplier: " + clazz.getName());
```

Comment:
We chose `info` in the test code for obvious reasons. Changing to `finest` makes debugging harder and noisier.